src/app/src/main/res/layout/activity_signup.xml

```diff
@@ -0,0 +1,135 @@
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="24dp"
+        android:layout_marginTop="24dp"
+        android:layout_marginEnd="24dp"
+        app:errorEnabled="true"
+        app:helperText="Last Name"
+        app:helperTextEnabled="true"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/signup_firstname_textInputLayout">
+
+        <com.google.android.material.textfield.TextInputEditText
+            android:id="@+id/signup_lastname_edit_text"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:hint="Last Name"
+            android:inputType="textPersonName" />
+    </com.google.android.material.textfield.TextInputLayout>
+
+    <com.google.android.material.textfield.TextInputLayout
+        android:id="@+id/signup_email_textInputLayout"
+        style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="24dp"
+        android:layout_marginTop="24dp"
+        android:layout_marginEnd="24dp"
+        app:errorEnabled="true"
+        app:helperText="Email"
+        app:helperTextEnabled="true"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/signup_lastname_textInputLayout">
+
+        <com.google.android.material.textfield.TextInputEditText
+            android:id="@+id/signup_email_edit_text"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:hint="Email"
+            android:inputType="textEmailAddress" />
+    </com.google.android.material.textfield.TextInputLayout>
+
+    <com.google.android.material.textfield.TextInputLayout
+        android:id="@+id/signup_phonenumber_textInputLayout"
+        style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="24dp"
+        android:layout_marginTop="24dp"
+        android:layout_marginEnd="24dp"
+        app:errorEnabled="true"
+        app:helperText="Phone Number"
+        app:helperTextEnabled="true"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/signup_email_textInputLayout">
+
+        <com.google.android.material.textfield.TextInputEditText
+            android:id="@+id/signup_phonenumber_edit_text"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:hint="Phone Number"
+            android:inputType="phone" />
```

Comment:
This seems like it would be the appropriate inputType given that it's `phone`, but is there a better alternative that is a little more trimmed down? It seems weird to be using a keyboard that has "N", commas, "Pause" and "Wait" keys, etc. I know we catch most of this with the regex (assuming it is correct), but we should guide the user as much as possible.

Reply:
That must be  you thing, my keyboard does not look like that. It looks like this:
https://pasteboard.co/Jzu1tbY.png

Reply:
Hit that bottom left button (#*)

Reply:
Hmmm. Did not catch that. Using simply number gives this: 
https://pasteboard.co/Jzu3Asf.png
This is much trimmed down. Some people enjoy using () and + in their international phone numbers though?

Reply:
Yeah they do. If there isn't a good in-between, for now I'd side on the less restrictive one. If we ever get around to formatting on focus loss (or better, formatting on text change so you can just press numbers and it gets formatted as you type), we could consider going to the more minimal keyboard. It's an extremely subtle detail though, don't think it matters too much anyways.